text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,7 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
+Hello! I'm duke.Duke.
```

Comment:
This was probably introduced by IntelliJ's search and replace - perhaps consider changing this to your actual expected text?